presto-main/src/main/resources/webapp/assets/query.js

```diff
@@ -1038,34 +1050,34 @@ let QueryDetail = React.createClass({
                                 </tr>
                                 <tr>
                                     <td className="info-title">
-                                        Client Address
+                                        Catalog
                                     </td>
                                     <td className="info-text">
-                                        { query.session.remoteUserAddress }
+                                        { query.session.catalog }
                                     </td>
                                 </tr>
                                 <tr>
                                     <td className="info-title">
-                                        Client Tags
+                                        Schema
                                     </td>
                                     <td className="info-text">
-                                        { query.session.clientTags.join(", ") }
+                                        { query.session.schema }
                                     </td>
                                 </tr>
                                 <tr>
                                     <td className="info-title">
-                                        Submission Time
```

Comment:
So `Submission Time` and `Completion Time` are removed intentionally?

Reply:
They have just been moved around